CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 1.2.1
+
+- Added support for windows projects on [Flutter 3.7.7](https://github.com/flutter/flutter/releases/tag/3.7.7)
```

Comment:
I checked and the changes are introduced in Flutter `3.7`, not `3.7.7`.

Reply:
Ok, have you checked if the latest flutter version introduced any changes again? Maybe checking that could come in quite handy, so that we can support the latest flutter version with the new version of this package.

Reply:
Nothing changed in Flutter 3.10 for windows, but changed in other platforms 👍🏼

I'll add support for those other changes and publish a new release ASAP.